universite-du-quebec-a-montreal.csl

```diff
@@ -738,28 +1151,65 @@
       </group>
     </layout>
   </citation>
-  <bibliography hanging-indent="true" et-al-min="8" et-al-use-first="6" et-al-subsequent-min="8" et-al-subsequent-use-first="6" et-al-use-last="true" entry-spacing="0" line-spacing="1">
+  <bibliography et-al-min="8" et-al-use-first="6" et-al-subsequent-min="8" et-al-subsequent-use-first="6" et-al-use-last="true" entry-spacing="0" hanging-indent="true">
```

Comment:
subsequent et al makes no sense for bibliography; remove.

Reply:
Understood.